### PR TITLE
travis: initial empty .travis.yml file

### PR DESCRIPTION
**Pull request by greut**
@chrisglass running the tests

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+dist: trusty
```
Comment (chrisglass) [inline .travis.yml]: Does travis not support xenial at least? trusty seems... ancient.

Comment (greut) [reply .travis.yml]: lmgtfy... no. :-D

Comment (greut) [reply .travis.yml]: Trusty is very fresh w.r.t to Travis, https://blog.travis-ci.com/2017-12-12-new-trusty-images-q4-launch

Comment (chrisglass) [reply .travis.yml]: Wow, ok :)

**Pull request merged**